src/hb-ot-hmtx-table.hh

```diff
@@ -51,6 +51,12 @@ struct LongMetric
   FWORD		sb; /* Leading (left/top) side bearing. */
   public:
   DEFINE_SIZE_STATIC (4);
+  LongMetric (unsigned ad,
+              unsigned side_bearing)
+  {
+    advance = ad;
+    sb = side_bearing;
+  }
```

Comment:
We actually don't currently have constructors in any of our font structs.  Previously that was a requirement of C++<11.  I still want to keep it that way.  So just assign to the members outside for now.  Thanks.